src/Validator/Csrf.php

```diff
@@ -10,27 +10,20 @@
 use function assert;
 use function explode;
 use function is_array;
+use function is_int;
 use function is_string;
 use function md5;
 use function random_bytes;
 use function sprintf;
 use function str_replace;
 use function strtr;
 
-/**
- * @psalm-type OptionsArgument = array{
- *     name?: non-empty-string,
- *     salt?: non-empty-string,
- *     session?: Container,
- *     timeout?: ?int,
- * }
- */
 final class Csrf extends AbstractValidator
```

Comment:
This whole class will probably need further refactoring such as deprecating and dropping all the public setters and getters

Reply:
Not suggesting that work should happen in this patch tho!